src/service/resources-impl.js

```diff
@@ -794,7 +794,10 @@ export class Resources {
             minTop = minTop == -1 ? box.top : Math.min(minTop, box.top);
           }
           request.resource./*OK*/changeSize(
-              request.newHeight, request.newWidth, request.callback);
+              request.newHeight, request.newWidth);
+          if (request.callback) {
+            request.callback(/* hasSizeChanged */true);
```

Comment:
Is this what you mean?
```
if (resize) {
  callback(true)
} else {
  if (isScroll) {
    continue;
  } else {
    callback(false);
  }
}
```

In this case I would need to add a variable for `isScroll`. Do you think it's worth it?